models/api_key.go

```diff
@@ -1,26 +1,19 @@
 package models
 
+import "time"
+
 type ApiKey struct {
 	Id             string
-	OrganizationId string
-	Hash           string
+	CreatedAt      time.Time
 	Description    string
+	Key            string
+	Hash           []byte
+	OrganizationId string
 	Role           Role
 }
 
-type CreatedApiKey struct {
-	ApiKey
-	Value string
-}
-
 type CreateApiKeyInput struct {
-	OrganizationId string
 	Description    string
+	OrganizationId string
 	Role           Role
 }
-
-type CreateApiKey struct {
```

Comment:
I'm simplifying things a bit here - the CreateApiKey basically _is_ the ApiKey. 
In a follow-up PR I might still reintroduce a notion of "CreatedApiKey" at the model level, but it's easier to bring it back when I need it.